### PR TITLE
Helm deployment

### PR DESCRIPTION
**Pull request by bszelag**
* helm deployment
* fixes in scrapper
* readme

### CODE REVIEW

File: helm-deployment/charts/minio-secrets-subchart/values.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+# values.yaml
```
Comment (MGniew) [inline helm-deployment/charts/minio-secrets-subchart/values.yaml:1]: Looks like very important file

File: helm-deployment/charts/rabbitmq-secrets-subchart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# values.yaml
```
Comment (MGniew) [inline helm-deployment/charts/rabbitmq-secrets-subchart/values.yaml:1]: Another empty file

File: helm-deployment/charts/scrap-me-subchart/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```
Comment (MGniew) [inline helm-deployment/charts/scrap-me-subchart/Chart.yaml:1]: Aren't those Chart.yaml files auto generated?

**Review by MGniew — CHANGES_REQUESTED** [review]

File: helm-deployment/charts/scrap-me-subchart/Chart.yaml (hunk 3 above)
Comment (bszelag) [reply helm-deployment/charts/scrap-me-subchart/Chart.yaml:1]: nope
and these files are required

**Review by MGniew — APPROVED** [review]

**Pull request merged**